feature/quicksettings/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (C) 2023 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <string name="quick_settings_front_camera_text">FRONT</string>
+    <string name="quick_settings_back_camera_text">BACK</string>
+
+    <string name="quick_settings_aspect_ratio_3_4">3:4</string>
+    <string name="quick_settings_aspect_ratio_16_9">16:9</string>
+    <string name="quick_settings_aspect_ratio_1_1">1:1</string>
+
+    <string name="quick_settings_flash_off">OFF</string>
+    <string name="quick_settings_flash_auto">AUTO</string>
+    <string name="quick_settings_flash_on">ON</string>
+
+    <string name="quick_settings_timer_off">OFF</string>
+    <string name="quick_settings_timer_3"></string>
+    <string name="quick_settings_timer_10"></string>
```

Comment:
Should these be removed if they're empty?

Reply:
These the value of these resources are empty. The resIds themselves are still needed for the logic